test/quantization/test_quantize_script.py

```diff
@@ -1169,12 +1169,58 @@ def _test_op_impl(self, module, data, quantized_op):
         model = quantize_script(model, qconfig_dict, _test_only_eval_fn, [data], inplace=False)
         FileCheck().check(quantized_op) \
                    .run(model.graph)
-
         # make sure it runs
         model(data[0][0])
 
         return model
 
+    @unittest.skipUnless(
```

Comment:
Do we need to skip QNNPACK?

Reply:
not sure if qnnpack works, my backward compatibility PR added qnnpack and the test failed

Reply:
When I enable the qnnpack locally, the tests pass

Reply:
I take it back -- the tests fail. Specifically line 950 in `observer.py` throws `RuntimeError: RuntimeError: Trying to create tensor with negative dimension -4398046511104`. I think that might be related to #36888